<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdOptDialog</class>
 <widget class="QWidget" name="CmdOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial/TCP Commands</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cBOpenCommands">
     <property name="toolTip">
      <string>Commands to be executed at startup</string>
     </property>
     <property name="text">
      <string>Commands at startup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="tEOpenCommands">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cBCloseCommands">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Commands at shutdown</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="tECloseCommands">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cBPeriodicCommands">
     <property name="text">
      <string>Periodic Commands : &lt;command&gt; # cycle (ms)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="tEPeriodicCommands"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Load...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Save...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
